From b5e1ac3c9fc7799b14d7684def9ec968194e36c7 Mon Sep 17 00:00:00 2001
From: "Christopher A. Oliver" <oliver@onion.private>
Date: Sat, 28 Jun 2014 18:18:44 -0400
Subject: [PATCH 0199/1936] Fix colors in bankUI buttons and partUI check boxes
 and text widgets to be reasonable for current NTK themes and standard fltk

---
 src/UI/BankUI.fl |  2 +-
 src/UI/PartUI.fl | 12 ++++++------
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/UI/BankUI.fl b/src/UI/BankUI.fl
index 08b4702d..85c343e3 100644
--- a/src/UI/BankUI.fl
+++ b/src/UI/BankUI.fl
@@ -89,7 +89,7 @@ refresh();} {}
     code {if (bank->emptyslot(nslot))
     color(46);
 else if (bank->isPADsynth_used(nslot)) 
-    color(26);
+    color(124);
 else 
     color(51);
 
diff --git a/src/UI/PartUI.fl b/src/UI/PartUI.fl
index b5c2b80c..78b904d4 100644
--- a/src/UI/PartUI.fl
+++ b/src/UI/PartUI.fl
@@ -858,12 +858,12 @@ if (part->Pkitmode==0) {
 part->kit[0].Ppadenabled=x;
 if (x==0) padeditbutton->deactivate();
      else padeditbutton->activate();}
-            tooltip {enable/disable PADsynth} xywh {215 255 80 20} box UP_BOX down_box DOWN_BOX color 51 selection_color 51 labelfont 1 labelsize 11
+            tooltip {enable/disable PADsynth} xywh {215 255 80 20} box UP_BOX down_box DOWN_BOX color 51 selection_color 0 labelfont 1 labelsize 11
             code1 {o->value(part->kit[0].Ppadenabled);}
           }
         }
         Fl_Group {} {
-          label ADDsynth
+          label ADsynth
           xywh {5 245 100 80} box ENGRAVED_FRAME labelfont 1
         } {
           Fl_Check_Button adsynenabledcheck {
@@ -872,7 +872,7 @@ if (x==0) padeditbutton->deactivate();
 part->kit[0].Padenabled=x;
 if (x==0) adeditbutton->deactivate();
      else adeditbutton->activate();}
-            tooltip {enable/disable ADsynth} xywh {15 255 80 20} box UP_BOX down_box DOWN_BOX color 51 selection_color 51 labelfont 1 labelsize 11
+            tooltip {enable/disable ADsynth} xywh {15 255 80 20} box UP_BOX down_box DOWN_BOX color 51 selection_color 0 labelfont 1 labelsize 11
             code1 {o->value(part->kit[0].Padenabled);}
           }
           Fl_Button adeditbutton {
@@ -892,7 +892,7 @@ if (x==0) adeditbutton->deactivate();
 part->kit[0].Psubenabled=x;
 if (x==0) subeditbutton->deactivate();
      else subeditbutton->activate();}
-            tooltip {enable/disable SUBsynth} xywh {115 255 80 20} box UP_BOX down_box DOWN_BOX color 51 selection_color 51 labelfont 1 labelsize 11
+            tooltip {enable/disable SUBsynth} xywh {115 255 80 20} box UP_BOX down_box DOWN_BOX color 51 selection_color 0 labelfont 1 labelsize 11
             code1 {o->value(part->kit[0].Psubenabled);}
           }
           Fl_Button subeditbutton {
@@ -919,14 +919,14 @@ if (x==0) subeditbutton->deactivate();
         Fl_Input {} {
           label {Author and Copyright}
           callback {snprintf((char *)part->info.Pauthor,MAX_INFO_TEXT_SIZE,"%s",o->value());}
-          xywh {5 60 385 50} type Multiline color 26 labelsize 10 align 5
+          xywh {5 60 385 50} type Multiline color 124 labelsize 10 align 5
           code0 {o->maximum_size(MAX_INFO_TEXT_SIZE);}
           code1 {o->value((char *) &part->info.Pauthor);}
         }
         Fl_Input {} {
           label Comments
           callback {snprintf((char *)part->info.Pcomments,MAX_INFO_TEXT_SIZE,"%s",o->value());}
-          xywh {5 125 385 90} type Multiline color 26 labelsize 11 align 5
+          xywh {5 125 385 90} type Multiline color 124 labelsize 11 align 5
           code0 {o->maximum_size(MAX_INFO_TEXT_SIZE);}
           code1 {o->value((char *) &part->info.Pcomments);}
         }
-- 
2.47.0

